nebula-components/components/_modal.scss

```diff
@@ -57,7 +57,7 @@
     right: 0;
     width: 40px;
     height: 40px;
-    background: $nb-color-black;
-    color: $nb-color-white;
+    background-color: $nb-color-alpha;
```

Comment:
I'd probably argue here that the close button would be the black colour, reason being that this modal could contain a call to action button that would likely be the alpha-colour and this would detract from that.